cmd/csi-driver/csi-driver.go

```diff
@@ -123,6 +123,13 @@ func csiCliHandler(cmd *cobra.Command) error {
 			return fmt.Errorf("Unable to configure iscsid service, err %v", err.Error())
 		}
 
+		// check if chapSecret is set in chap.json
+		driver.ChapUsername, driver.ChapPassword, err = tunelinux.GetIscsiChapSettings()
```

Comment:
i am thinking if we need to add CHAP login in tunelinux at all. Since it has more of set/get recommendations pattern, chap doesn't fit into that as there is no recommendation for this. Can we directly fetch these from ENV params with node plugin and populate hpenodeinfos with that?

Reply:
are you suggesting we don't need the additional parameters configMap as well? I am fine with that approach but we already have quite a few env variables to overload the csi driver. 
Additionally this fits well with the iscsid parameters we deal with.
@rgcostea what do you recommend?

Reply:
CHAP settings are a property of iSCSI configurations.  Can you describe what the purpose of TuneLinux is exactly?  Why aren't we using a ConfigMap in that case?  Historical reasons?  What are the pros and cons of each approach?